cli/runtime/plugin/info_test.go

```diff
@@ -43,16 +42,28 @@ func TestInfo(t *testing.T) {
 		DocURL:      "https://docs.example.com",
 		Hidden:      false,
 	}
-	expected, err := json.Marshal(descriptor)
-	if err != nil {
-		t.Error(err)
-	}
 
 	infoCmd := newInfoCmd(&descriptor)
 	err = infoCmd.Execute()
 	w.Close()
 	assert.Nil(err)
 
 	got := <-c
-	assert.Equal(fmt.Sprintf("%s\n", expected), string(got))
+
+	expectedInfo := cliapi.PluginInfo{
+		PluginDescriptor: descriptor,
+	}
+
+	gotInfo := &cliapi.PluginInfo{}
+	err = json.Unmarshal(got, gotInfo)
+	if err != nil {
+		t.Error(err)
+	}
+
+	assert.Equal(expectedInfo.Name, gotInfo.Name)
+	assert.Equal(expectedInfo.Description, gotInfo.Description)
+	assert.Equal(expectedInfo.Version, gotInfo.Version)
+	assert.Equal(expectedInfo.BuildSHA, gotInfo.BuildSHA)
+	assert.Equal(expectedInfo.DocURL, gotInfo.DocURL)
+	assert.Equal(expectedInfo.Hidden, gotInfo.Hidden)
```

Comment:
Shouldn't we test for the new `PluginRuntimeVersion` here?

Reply:
Yes. I should have added one. But I was thinking of adding it to specify the value returned is not empty instead of comparing it with static value. The main reason is we do not want to update this test each time we bunch the `PluginRuntimeVersion`.

Reply:
Added test to verify the `PluginRuntimeVersion` is not empty when returned with info.